nn_play.py

```diff
@@ -82,17 +83,18 @@ def load_resource(self):
         print('==== successfully restored ====')
 
     def get_current_state(self):
+        localfile = "state_{}.png".format(str(uuid.uuid4()))
         if self.phone == 'Android':
-            os.system('C:\\projects\\Wechat_AutoJump\\adb\\adb.exe shell screencap -p /sdcard/1.png')
-            os.system('C:\\projects\\Wechat_AutoJump\\adb\\adb.exe pull /sdcard/1.png state.png')
+            os.system("C:\\projects\\adb\\adb.exe shell screencap -p /sdcard/1.png")
```

Comment:
move  "C:\\projects\\adb\\adb.exe"  to global setting

Reply:
the same as  [l89](https://github.com/lljieying/WeJump/pull/2/files#diff-3d98ccf4218269571ea7122f1d467f0aR89) and   [l167](https://github.com/lljieying/WeJump/pull/2/files#diff-3d98ccf4218269571ea7122f1d467f0aR167)